docs/overview/getting-started/avax.md

```diff
@@ -0,0 +1,26 @@
+# AVAX
+
+AVAX is the native utility token of Avalanche. It’s a hard-capped, scarce asset that is used to 
+pay for fees, secure the platform through staking, and provide a basic unit of account between the
+multiple Subnets created on Avalanche. `1 nAVAX` is equal to `0.000000001 AVAX`.
+
+## Utility
+
+AVAX is a non-renewable resource (analogous to oil) that's used to power the Avalanche network.
```

Comment:
Can we remove the reference to `non-renewable` and `analogous to oil` ? I understand the intention to make it sound like a scarce resource since that has a positive connotation in crypto, but think we can do that without referencing non-renewable energy which has a more negative connotation.

Reply:
Sure